modules/installation-configuration-parameters.adoc

```diff
@@ -1223,6 +1223,54 @@ Additional Azure configuration parameters are described in the following table:
 |Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance. If instance type of compute machines support `Accelerated` networking, by default, the installer enables `Accelerated` networking, otherwise the default networking type is `Basic`.
 |`Accelerated` or `Basic`.
 
+|`compute.platform.azure.type`
+|Defines the azure instance type for compute machines.
+|String
+
+|`compute.platform.azure.zones`
+|The availability zones where the installation program creates compute machines.
+|String list
+
+|`controlPlane.platform.azure.type`
+|Defines the azure instance type for control plane machines.
+|String
+
+|`controlPlane.platform.azure.zones`
+|The availability zones where the installation program creates control plane machines.
+|String list
+
+|`platform.azure.defaultMachinePlatform.encryptionAtHost`
+|Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached and un-managed disks on the VM host. This is not a prerequisite for user-managed server-side encryption.
+|`true` or `false`. The default is `false`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.name`
+|The name of the disk encryption set that contains the encryption key from the installation prerequisites.
+|String, for example, `production_disk_encryption_set`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.resourceGroup`
+|The name of the Azure resource group that contains the disk encryption set from the installation prerequisites.  To avoid deleting your Azure encryption key when the cluster is destroyed, this resource group must be different from the resource group where you install the cluster. This value is only necessary if you intend to install the cluster with user-managed disk encryption.
```

Comment:
```suggestion
|The name of the Azure resource group that contains the disk encryption set from the installation prerequisites. To avoid deleting your Azure encryption key when the cluster is destroyed, this resource group must be different from the resource group where you install the cluster. This value is necessary only if you intend to install the cluster with user-managed disk encryption.
```

Per IBM Style, place "only" as close to the word or phrase that it modifies as possible: https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only